Jenkinsfile

```diff
@@ -0,0 +1,98 @@
+#!/bin/env groovy
+
+@Library('cliqz-shared-library@vagrant') _
+
+node('mac-mini-ios') {
+    writeFile file: 'Vagrantfile', text: '''
+    Vagrant.configure("2") do |config|
+        config.vm.box = "ios-xcode9.2"
+        
+        config.vm.define "priosx92" do |priosx92|
+            priosx92.vm.hostname ="priosx92"
+            
+            priosx92.vm.network "public_network", :bridge => "en0", auto_config: false
+            priosx92.vm.boot_timeout = 900
+            priosx92.vm.provider "vmware_fusion" do |v|
+                v.name = "priosx92"
+                v.whitelist_verified = true
+                v.gui = false
+                v.memory = ENV["NODE_MEMORY"]
+                v.cpus = ENV["NODE_CPU_COUNT"]
+                v.cpu_mode = "host-passthrough"
+                v.vmx["remotedisplay.vnc.enabled"] = "TRUE"
+                v.vmx["RemoteDisplay.vnc.port"] = ENV["NODE_VNC_PORT"]
+                v.vmx["ethernet0.pcislotnumber"] = "33"
+            end
+            priosx92.vm.provision "shell", privileged: false, run: "always", inline: <<-SHELL#!/bin/bash -l
+                set -e
+                set -x
+                rm -f agent.jar
+                curl -LO #{ENV['JENKINS_URL']}/jnlpJars/agent.jar
+                java -version
+                nohup java -jar agent.jar -jnlpUrl #{ENV['JENKINS_URL']}/computer/#{ENV['NODE_ID']}/slave-agent.jnlp -secret #{ENV["NODE_SECRET"]} &
+            SHELL
+        end
+    end
+    '''
+
+    def jobStatus = ""
+
+    vagrant.inside(
+        'Vagrantfile',
+        '/jenkins',
+        4, // CPU
```

Comment:
4 cpu is quite a lot :D have you benchmarked how number of cpu affect build time?

Reply:
@chrmod When we are running tests on the Simulator, it is better to use 4 CPUs to make it finish faster.
As of now reduced them to 2 and we can benchmark them at a later point of time when the tests are ready.